quests/conditional_statements.md

```diff
@@ -190,10 +190,10 @@ template with the same `webrick` default. The Pasture appication passes any
 settings under the `:sinatra_settings:` key to Sinatra itself.
```

Comment:
Need to remote the reference in these instructions too.

"Add the `$sinatra_server` variable to the params block at the beginning of the template."